index.html

```diff
@@ -44,5 +44,115 @@ <h2 class="social">
+          <li><img src="icon/javascript_tag.png" alt="javascript"></li>
+        </ul>
+        <button type="button" class="button">See Project</button>
+      </article>
+      <article class="proj">
+        <img src="img/snapshot2.png" class="proj_img" alt="Multi-Post Stories">
+        <h2 class="proj_title">Multi-Post Stories</h2>
+        <div class="proj_subt d-flex">
+          <div class="black">CANOPY</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">Back End Dev</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">2015</div>
+        </div>
+        <p class="proj_desc black">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="proj_lang">
+          <li><img src="icon/html_tag.png" alt="html"></li>
+          <li><img src="icon/css_tag.png" alt="css"></li>
+          <li><img src="icon/javascript_tag.png" alt="javascript"></li>
+        </ul>
+        <button type="button" class="button">See Project</button>
+      </article>
+      <article class="proj">
+        <img src="img/snapshot3.png" class="proj_img" alt="Tonic">
+        <h2 class="proj_title">Tonic</h2>
+        <div class="proj_subt d-flex">
+          <div class="black">CANOPY</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">Back End Dev</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">2015</div>
+        </div>
+        <p class="proj_desc black">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="proj_lang">
+          <li><img src="icon/html_tag.png" alt="html"></li>
+          <li><img src="icon/css_tag.png" alt="css"></li>
+          <li><img src="icon/javascript_tag.png" alt="javascript"></li>
+        </ul>
+        <button type="button" class="button">See Project</button>
+      </article>
+      <article class="proj">
+        <img src="img/snapshot4.png" class="proj_img" alt="Multi-Post Stories">
+        <h2 class="proj_title">Multi-Post Stories</h2>
+        <div class="proj_subt d-flex">
+          <div class="black">CANOPY</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">Back End Dev</div>
+          <img src="icon/counter.png" alt="counter">
+          <div class="proj_ssubt2">2015</div>
+        </div>
+        <p class="proj_desc black">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="proj_lang">
+          <li><img src="icon/html_tag.png" alt="html"></li>
+          <li><img src="icon/css_tag.png" alt="css"></li>
+          <li><img src="icon/javascript_tag.png" alt="javascript"></li>
+        </ul>
+        <button type="button" class="button">See Project</button>
+      </article>
+      <article class="about">
```

Comment:
- [ ] It would be better if you use `div` tags instead of `article` tags in order to make your code more Semantic. For this little thing, I don't want you to make another request. Before merging, please change these `article` tags with div` 🙏.

Reply:
Ok, thank you for the tip.